style.css

```diff
@@ -1 +1,357 @@
+@import /*  Add your styling here */
+  url('https://fonts.googleapis.com/css2?family=Cookie&family=Montserrat&display=swap');
 /*  Add your styling here */
+:root {
+  --green: #15f1a7;
+  --yellow: #edf50c;
+  --green-light: #8be5f3;
+  --red-light: #ffcab0;
+  --blue-light: #83cee0;
+  --purple: #ea6af6;
+  --pink: #ef61cb;
+  --black: #000;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: repeat(3, 1fr);
+  grid-column-gap: 0px;
+  grid-row-gap: 0px;
+  background-color: var(--green-light);
+  height: 100vh;
+  font-family: "Montserrat", sans-serif;
+}
+/* header */
+header {
+  grid-area: 1 / 1 / 2 / 2;
+  background-color: var(pink);
+  /* header grid */
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: repeat(2, 1fr);
```

Comment:
This does align the title at the bottom of the header correctly but there's a simpler way; set display: flex on the header, and then use align-self: flex-end on the `p` which will vertically align it to the bottom of the container.